canonical-web-and-design/anbox-cloud.io
# Invitation code

## Done

Entire login flow.
Pending: waiting for anbox-cloud API to release an endpoint to test the token.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Turn on VPN
- Login with new user without invitation code ->  "http://0.0.0.0:8043/demo" it will fail
- Login with new user with invitation code -> "http://0.0.0.0:8043/demo?invitation_code=XXXX" where XXXX needs to be generated by anbox-cloud API manually.
